config.ru

```diff
@@ -0,0 +1,12 @@
+require 'sidekiq'
+require 'sidekiq-scheduler/web'
```

Comment:
sidekiq and sidekiq-scheduler are being required in psublib_traject. are these needed here?

Reply:
only the /web requires are in config now